### ISSUE TITLE
Export report as DOCX or PDF

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#573]
Currently you can export a report a a CSV, which is good for raw data but has no formatting or grouping.

And you can copy and paste the report results into word, but that doens't work very well because there's lots of kruft attached to mobile numbers etc.

It would handy to be able to export the report in a fairly plain DOCX or PDF.  And/or have a plain "printable" version like we do for rosters and run sheets.

**Comment by tim-pearce** [comment]
See pull request #448.
I've been using this all year now and have made a few minor adjustments to my local copy.

**Comment by tbar0970** [comment]
I've been very slow to merge that pull request, I'm sorry. 
This FR is slightly different in that it's not a merge but just a dump of the data on the screen.
But the same library might be useful here - thanks for the pointer.

**Comment by tim-pearce** [comment]
Yes, the same library will work. The library I use is primarily an XML/HTML processor. The handling of documents is an add on which works on compressed XML, which is what these documents are.
It enables you to create templates. One thing I would possibly change (or add) would be for the templates to be on the server. I haven't had a problem with the templates being offline because (1) it's just me and (2) the templates are on a Nextcloud server.

**Comment by tbar0970** [comment]
+1
